funnel/templates/event.html.jinja2

```diff
@@ -80,7 +80,7 @@
               <td class='js-searchable' data-th="Company">{{ company }}</td>
               <td class='buttongrp-column'>
                 <ul class="mui-list--unstyled">
-                  {{#if isAdmin || isConcierge}}
+                  {{#if isEditor || isConcierge}}
```

Comment:
For consistency, these two should be `isProjectEditor` and `isProjectConcierge`.